client/plots/disco/data/DataMapper.ts

```diff
@@ -67,23 +67,48 @@ export default class DataMapper {
 		return aPos - bPos
 	}
 
-	constructor(settings: Settings, reference: Reference, sample: string, prioritizedGenes: Array<string> = []) {
+	private prioritizeGeneLabelsByGeneSets: boolean
+
+	constructor(
+		settings: Settings,
+		reference: Reference,
+		sample: string,
+		prioritizedGenes: Array<string> = [],
+		prioritizeGeneLabelsByGeneSets: boolean
```

Comment:
@aacic shouldn't this configuration be tracked as `settings.prioritizeGeneLabelsByGeneSets`, and not as an independent argument?

Reply:
Good catch. It was already in settings.prioritizeGeneLabelsByGeneSets, I've just made a duplicate here. I've removed it. You can review it again.